integration-templates/hubspot/hubspot-contacts.ts

```diff
@@ -1,12 +1,36 @@
 import type { NangoSync, HubspotContact } from './models';
 
 export default async function fetchData(nango: NangoSync) {
-    const query = `properties=firstname,lastname,email`;
+    //handle query properties well
+    const query = `firstname,lastname`;
```

Comment:
Why was email removed? Can the query structure also be moved to use an array and then join it:

 const properties = ['firstname', 'lastname', 'email', 'phone', 'hs_marketable_status', 'annualrevenue']; 
 
{ endpoint: "/crm/v3/objects/contacts", params: {  properties: properties.join(',')  } }